packages/transit-vehicle-overlay/src/index.story.js

```diff
@@ -229,7 +231,7 @@ function realtimeExample(fetchVehicles, fetchPattern, markers) {
   // give action to the popup vehicle tracking button
   VehiclePopup.defaultProps.setTracked = (vehicle, isTracked) => {
     updateTrackedVehicle(vehicle, isTracked);
-    if (!isTracked) recenter(null, 0, 0); // clear out cached coords ... recenter on recently untracked vehicle
+    if (!isTracked) recenter(null, 0, true, 0); // clear out cached coords ... recenter on recently untracked vehicle
```

Comment:
Same as https://github.com/opentripplanner/otp-ui/pull/255/files#r648383383, I am not sure to understand the change in this function call.